repo.go

```diff
@@ -240,14 +191,7 @@ func (r *Repo) GetAllFiles(subPath string, ignoreSymlinks bool) (map[string]*Fil
 			continue
 		}
 
-		fileLog, err := r.getFileLog(path)
-		if err != nil {
-			return nil, fmt.Errorf("unable to get log for %s: %v", path, err)
-		}
-		files[path] = &File{
-			file: file,
-			Log:  fileLog,
-		}
+		files[path] = &File{file: file}
```

Comment:
Why aren't we adding the commit in here? We could change `getAllFiles` to return the `File` object including commit and then the filter would just add the pointers to the new list rather than constructing new `File`s, WDYT?

Reply:
Because I missed adding it. Yeah, that sounds more reasonable, see the latest commit.